packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -2449,6 +2449,24 @@ describe('CliRepl', function () {
         await waitCompletion(cliRepl.bus);
         expect(output).to.include('res.acknowledged');
       });
+
+      it('completes only collection names that do not include control characters', async function () {
+        if (!hasCollectionNames) return;
```

Comment:
very nit: we could `this.skip()` instead

Reply:
Sure, done!